src/User/RegistrationForm.jsx

```diff
@@ -15,9 +15,24 @@ const RegistrationForm = () => {
     lastName: "",
     email: "",
     occupation: "",
+    managerId: "",
     username: "",
     password: ""
   });
+  const [managers, setManagers] = useState([]);
+
+  const fetchData = async () => {
+    try {
+      const userList = await getUserList();
```

Comment:
Why are managers all users?

Reply:
To implement manager profiles the BE users table is being altered to include a `managerId` column which connects each user to another user's `id`. Therefore managers are users which are assigned to their role in the system by users selecting their name when they register. So if a manager has registered with the app but none of their employees have registered yet they wouldn't be searchable as a manager despite being one. So essentially we have to use get all users in this situation as otherwise some managers might be missed out.

Apologies for the wall of text 😄 let me know if it doesn't make any sense!

Reply:
Yeh I'm a bit confused on what this PR is for, its a list of users?

Reply:
I've renamed the PR, since it took the commit message as it's name, which wasn't very descriptive.

Essentially this PR is for adding in a `<select>` on the registration page which allows you to select your manager from a list of users.

Reply:
And the associated API request

Reply:
So am I right in assuming if this is deployed and we have 1000's of users the options will be endless?

Reply:
If it's left in the current state yes, when we discussed this in Sprint Planning we decided to do it this way as a placeholder and then in the future add something like autocomplete/search functionality. I think @GBurley-Jones and @sophiambolah will probably be better at answering questions about that.

Reply:
So, from what I've gathered, it sounds like the ticket shouldn't really be done yet. What you've done is fine, but just something to think about going forward is that it's generally not worth implementing something as a placeholder until its actually ready to go. There's several reasons for this but one of them can be that product might get the impression that it's working and keep it in, or you might get busy with something else and not come back to it.

I've spoken to a few people about how you're implementing this feature and sounds like you want to use elastic search for something that should be a simple SQL to check if your database has a certain string in it for the names of the users? 

Just something you might want to bring up with the team - think elastic search is major overkill for a search on user name